packages/core/src/App/Components/Layout/Header/dtrader-v2/real-signup-banner-dtrader-v2.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import classNames from 'classnames';
+import 'Sass/app/_common/components/banner-dtrader-v2.scss';
```

Comment:
it's better to align the name of the css file with the name of the component file: :)  
```suggestion
import 'Sass/app/_common/components/real-signup-banner-dtrader-v2.scss';
```

Reply:
Oh, I forgot to rename this scss file after I renamed the component, thank you for noticing)